mkdocs.yml

```diff
@@ -16,6 +16,7 @@ nav:
   - Core utilities:
       - core/logging.md
       - core/metrics.md
+      - core/metrics-v2.md
```

Comment:
Please add a root level for `Metrics` and make metrics v1 and the default link.

Reply:
![image](https://github.com/user-attachments/assets/cd6049e1-68a4-4bd8-9242-6c71e7ba28ee)